src/app/shared/components/student-profile-apprenticeship-company/student-profile-apprenticeship-company.component.html

```diff
@@ -45,8 +48,12 @@
 <div *ngIf="company.jobTrainer" class="subentry">
   {{ 'shared.profile.job-trainer' | translate }}:
   <address>
-    {{ company.jobTrainer.Lastname }}
-    {{ company.jobTrainer.Firstname }}<br />
+    <ng-container
+      *ngIf="company.jobTrainer.Lastname && company.jobTrainer.Firstname"
+    >
+      {{ company.jobTrainer.Lastname }}
+      {{ company.jobTrainer.Firstname }}<br />
+    </ng-container>
```

Comment:
Könnte ev. auch nur der Nachname gesetzt sein?

Reply:
Ich weiss es nicht. Das scheint mir nicht wahrscheinlich.

Dieselbe Frage würde sich dann auch bei `apprenticeshipManager` und `person` stellen.

Reply:
Es wäre wohl etwas robuster dies vorzusehen:

```
    <ng-container *ngIf="company.jobTrainer.Lastname || company.jobTrainer.Firstname"
    >
      {{ company.jobTrainer.Lastname || '' }}
      {{ company.jobTrainer.Firstname || '' }}<br />
    </ng-container>
```

Aber ist wohl nicht so wichtig...

Reply:
Find ich gut und habe ich für die Fälle in der ApprenticeshipCompany-Komponente so übernommen.